gradlew

```diff
@@ -1,172 +0,0 @@
-#!/usr/bin/env sh
```

Comment:
This could be an environment issue on my end (I use intellij for Flutter projects), but without gradlew and gradlew.bat, intellij doesn't detect that the project uses gradle to build and the project doesn't get built with changes to the java side of the plugin. This may be a more general gradle question, but what's the reasoning behind removing this?

Reply:
I'm using Android Studio, which works fine without this file. I'll just add these back in